src/components/PostEdit/components/PostEditEditor/PostEditEditor.tsx

```diff
@@ -97,12 +97,13 @@ const PostEditEditor = ({ onEdit, postData }: PostEditEditorProps) => {
           return
         }
       })
+      return
```

Comment:
else-if를 사용하고 얼리리턴을 줄이면 코드를 더 줄일 수 있을 것 같아요.
예를들면 이렇게..?

```suggestion
    if (postData.postId === "newPost") {
      createPost(postData).then((res) => {
        if (res) {
          showComplete()
        } else {
          setAlertMessage(POST_EDIT_ERROR_MESSAGE.SUBMIT_ERROR_NEW_POST)
          showAlert()
        }
      })
    } else if (postData.postId) {
      updatePost(postData).then((res) => {
        if (res) {
          showComplete()
        } else {
          setAlertMessage(POST_EDIT_ERROR_MESSAGE.SUBMIT_ERROR_UPDATE_POST)
          showAlert()
        }
      })
    }
```

Reply:
얼리리턴을 통해서 해당 조건 내부에서 어떤일을 하는지 문장 단위로 보여질 수 있는 점이 가독성이 더좋다고 느껴져서 위와같이 얼리리턴을 사용하였습니다